how-to/debug-buildfarm-builder.rst

```diff
@@ -9,12 +9,18 @@ Therefore, to access the builder, one should either:
 
 * access it during the duration of a build.
 
-* set the builder to ``manual`` after starting a build so that it doesn't
+* set the builder to ``disabled`` after starting a build so that it doesn't
   automatically trigger the reset when the build finishes.
 
+.. note::
 
-Set a builder to manual
-~~~~~~~~~~~~~~~~~~~~~~~
+   You should wait until you see some output in the build log before you set
+   the builder to ``disabled``, otherwise the build might not get dispatched
+   properly.
+
+
+Set a builder to disabled via UI
```

Comment:
The section above still talks about switching to a manual-mode instead of disabled. Should we simply remove it and only mention the CLI approach which is the one we would usually use?

Reply:
Indeed - I just checked and it looks like via the UI you cannot set builder to disabled, but only to manual. I'll remove the UI section.